packages/usdk/packages/upstreet-agent/packages/react-agents/classes/discord-manager.ts

```diff
@@ -390,6 +444,7 @@ export class DiscordBot extends EventTarget {
             method: 'say',
             args: {
               text: formattedMessage,
+              messageId,
```

Comment:
A few issues:

- This is discord specific; it's not directly related to agent `say` messages. It's just metadata about the client connection that the conversation is bound to.
- `messageId` will be rendered into the prompt, confusing the LLM.
- Are we strongly typing the messages + args? It would be nice if this were all validated rather than us adding keys like this. This is stored in the database too, so lack of validation can hit us further down the road when we are reading this data back and it has an unexpected format.

I think the ideal here would be:
1) we store some "metadata" key that contains this client specific information which is not generally applicable to all messages
2) we specifically do _not_ render this metadata into the LLM prompts
3) we strongly type messages to check the shape so we avoid this kind of issue in the future